.travis.yml

```diff
@@ -20,6 +20,7 @@ install:
   - pip install --upgrade ansible==2.8.* docker
```

Comment:
Please update travis.ci according to last ADK templates. Python3 should be used.

Reply:
Done.